frontend/components/vault/VaultLiquidationTransactionFlow.vue

```diff
@@ -0,0 +1,147 @@
+<template>
+    <div>
+        <TextBlock title="Vault Liquidation transaction" />
+        <Alert v-if="wasLiquidated" show-icon type="warning">
+            <div slot="message">
+                <p>This vault has been liquidated into a collateral auction</p>
+                <div class="flex justify-end mt-2">
+                    <nuxt-link :to="auctionLink">
+                        <Button> View collateral auction {{ vaultTransaction.auctionId }} </Button>
+                    </nuxt-link>
+                </div>
+            </div>
+        </Alert>
+        <VaultLiquidationTransactionTable v-if="!wasLiquidated" class="my-4" :vault-transaction="vaultTransaction" />
+        <TextBlock class="mt-4">
+            <template v-if="wasLiquidated">
+                This vault was liquidated <TimeTill :date="vaultTransaction.liqudiationDate" /> in the transaction
+                <FormatAddress :value="vaultTransaction.transactionHash" />.
+            </template>
+            <template v-else>
+                Please note, the
+                <Explain text="transaction fee" width="350px">
+                    <TransactionFeesTable
+                        :fees="{ LiquidationFee: vaultTransaction.transactionFeeLiquidationEth }"
+                        :combined-fees-eth="vaultTransaction.transactionFeeLiquidationEth"
+                    />
+                </Explain>
+                is a suggested value based on the current gas prices on the market; the “potential net profit” is also
+                approximate, since it is extrapolated from the exchange rates and may change during the transaction. If
+                you’re executing transactions with other participants at the same time, the one who pays a higher
+                transaction fee has more chances to receive the incentive. In case your transaction will be rejected,
+                it will only result in the loss of the transaction fee.
+            </template>
+        </TextBlock>
+        <div v-if="!wasLiquidated" class="mb-4">
```

Comment:
There is no space between the text and the panels

```suggestion
        <div v-if="!wasLiquidated" class="my-4">
```